tools/Az.Tools.Predictor/Az.Tools.Predictor/Commands/EnableAzPredictor.cs

```diff
@@ -18,7 +18,7 @@
 namespace Microsoft.Azure.PowerShell.Tools.AzPredictor
 {
     /// <summary>
-    /// A cmdlet that enables Az Predictor with default settings.
+    /// <para type="description"> Use this cmdlet to enable Az Predictor and start recieving suggestions</para>
```

Comment:
it requires synopsis.

Reply:
Here is example https://www.red-gate.com/simple-talk/dotnet/software-tools/documenting-your-powershell-binary-cmdlets/

Reply:
Added synopsis